.env.template

```diff
@@ -2,6 +2,10 @@ PINECONE_API_KEY=your-pinecone-api-key
 PINECONE_ENV=your-pinecone-region
 OPENAI_API_KEY=your-openai-api-key
 TEMPERATURE=1
+TOP_P=1
```

Comment:
Can you please prefix these new keys OPENAI_ ?

Reply:
Sure, i actually did it at first but remove it for some reason

Reply:
Please approve asap, as I dont want to keep solving merge conflicts..